src/test/java/com/fewlaps/quitnowemailsuggester/StringUtilsTest.java

```diff
@@ -34,4 +37,53 @@ public void shouldReturnHeo() {
     public void shouldReturnAaabxx() {
         assertEquals("aaabxx", su.replaceLast("aaabbb", "bb", "xx"));
     }
+
+
+    @Test
+    public void shouldReturnNullForNull() throws InvalidEmailException {
+        String[] parts = su.getParts(null);
+        assertNull(parts);
+    }
+
+    @Test
+    public void shouldReturnArrayWithxZeroElementsFromEmpty() throws InvalidEmailException {
+        String[] elements = su.getParts("");
```

Comment:
You talked about *parts* six lines before. Why not calling it *parts* too, instead of *elements*?